README.md

```diff
@@ -0,0 +1,40 @@
+# saturn-client
+Python library for interacting with [Saturn Cloud](https://www.saturncloud.io/) API.
+
+## Connect
+
+```python
+from saturn import SaturnConnection
+
+# From outside of Saturn
+conn = SaturnConnection(url="<SATURN_URL>", api_token="<API_TOKEN>")
+
+# From inside of Saturn
+conn = SaturnConnection()
+```
+
+## Create a Project
+
+The minimal acceptable input is the name of the project:
+
+```python
+project = conn.create_project(name="my-project")
+```
+
+But all the other settings that are available in the UI can also be passed in:
+
+```python
+project = conn.create_project(
+    name="my-project"
+    description="My new project - created from outside of Saturn programatically!",
+    image_uri="saturncloud/saturn-gpu:2020.11.30",
+    start_script="pip install git+https://github.com/saturncloud/dask-saturn.git@main",
+    environment_variables={"DATA_URL": "s3://my-bucket/data"},
+    working_dir="/home/jovyan/project",
+    workspace_settings={
+        "size": "large",
+        "auto_shutoff": "Never",
+        "start_ssh": True,
```

Comment:
since this is an expensive option, I'd prefer to default it to `False` in this introductory documentation

Reply:
Hmm I disagree. This would be a nice way of interacting from fully outside of Saturn. Plus this snippet isn't really intended to be copy/pasted, it is more of an indication of what is possible.

Reply:
I agree with James on this one. This setting spins up a new ELB for each Jupyter server. I don't think it's something we should blanket encourage people to enable. If a user wants to enable this, they should really know what they are doing. Not to mention, enabling this isn't even useful unless the user has also configured an SSH public key credential, which I believe is not an option in this client yet.

Reply:
Ok. Overruled! :)